src/config.c

```diff
@@ -3276,6 +3276,7 @@ standardConfig static_configs[] = {
     createIntConfig("timeout", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.maxidletime, 0, INTEGER_CONFIG, NULL, NULL), /* Default client timeout: infinite */
     createIntConfig("replica-announce-port", "slave-announce-port", MODIFIABLE_CONFIG, 0, 65535, server.replica_announce_port, 0, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("tcp-backlog", NULL, IMMUTABLE_CONFIG, 0, INT_MAX, server.tcp_backlog, 511, INTEGER_CONFIG, NULL, NULL), /* TCP listen backlog. */
+    createIntConfig("mptcp", NULL, IMMUTABLE_CONFIG, 0, INT_MAX, server.mptcp, 0, INTEGER_CONFIG, NULL, NULL),               /* Multipath TCP. */
```

Comment:
Maybe it makes sense to have a dedicated config for replication.

Reply:
Yes, I have the same opinion in the next step.

Because server and client benchmark MPTCP support could be tested easily, we can know the test result in the commit message.

Reply:
@xbasel 

This is the first step - server side supports MPTCP. If this is merged into valkey, then I'll support:
- explicit MPTCP config for replication. This will be helpful for replication data synchronization over a long distance.
- support libvalkey. After valkey uses libvalkey instead of hiredis, the client tools will support MPTCP easily. https://github.com/valkey-io/valkey/pull/1802 is ready to review, this will help MPTCP to support MPTCP with a few code changes.

Reply:
Right, fix it and append a commit. Thanks!